### DISCUSSION TITLE
Turning complaints into wins - Chewing the Fat 📈

### DISCUSSION THREAD

**Discussion by lukecookssw** [SSWConsulting/SSW.Rules.Content#5241] [Chewing the Fat]
Contributors: @adamcogan @tiagov8 @lukecookssw 

Let's talk about this rule: [Do you make complaints a positive experience?](https://www.ssw.com.au/rules/make-complaints-a-positive-experience/)

**Comment by AshAnil** [comment]
Nice rule ✅. It emphasizes the importance of listening actively and building stronger relationships with customers. 
Ideally, maintain a smile on your face when talking to an unhappy/frustrated customer in person.

**Comment by tiagov8** [comment]
I just added a PR to include 1 extra point to the rule: **Don't take it personally!**

I find this very important, as it's possibly the greatest cause of broken relationships

**Comment by matt-goldman** [comment]
Acknowledge the problem. This is the most important thing, followed by be active rather than defensive. Make it you and the customer vs the problem rather than you vs the customer.

**Comment by ozdemir-mehmet** [comment]
Good non-tech topic, relevant to devs.

**Comment by bahjatalaadelssw** [comment]
Listen, find out the original point of contact and say "Let's get this sorted now" It's important to let the customer know that the solution process has started because that will take the edge off being ignored which causes unnecessary anger.

**Comment by jakebayliss** [comment]
SETO - Show Empathy + Take Ownership. I think these are super important, the customer wants to know that you acknowledge their concern and will action it quickly

**Comment by kikibianc** [comment]
Does this work for internal complaints too? I think following these for internal complaints would work wonders, too :)

**Comment by AlexRothwell** [comment]
I think this can be viewed as an opportunity to iterate on your product/process. Viewing it as an opportunity should hopefully help to take it less personally and reach a more productive result

**Comment by AntPolkanov** [comment]
For complex problems, I think it's very important to demonstrate the client that you are willing to solve the issue rather than just ticking the box.

**Comment by UlyssesMaclaren** [comment]
1. Remember that it's you and the client working together against the situation... not you against the client. 
2. Always work forwards not backwards. work towards a solution rather than fixating on the past (what went wrong and who's fault it was)

**Comment by tombui99** [comment]
These are some very good tips, there's no point in arguing with the client just to save face but always try to propose a solution and action it as soon as possible.

**Comment by camillars** [comment]
The first thing to do is to recognize the problem. Instead of becoming defensive, try to see it as a shared problem between you and the customer, rather than a conflict between the two of you.

**Comment by cvionnet** [comment]
It is good to remind to not take it personally, even it is quite difficult when you face the problem. This can affect all other steps: you stop listening and can't think properly.
I also learned about following up when the problem is solved.

**Comment by JeanThirion** [comment]
Crazy important topic. As consultants, we are exposed to issues on a daily basis really. And it's up to us to make the extra effort to turn a hairy situation into a win. 

Knowing when it's out of your league, i.e. when to raise the issue to the account manager or above is very important too. If you can manage yourself, great, otherwise do not hesitate to raise a flag.

**Comment by Ben0189** [comment]
I learned from the rule - understand the issue, bring in relevant people if needed, and follow up the solution

**Comment by amankumarrr** [comment]
By paying close attention to the specifics and avoiding anything that a client has expressed dissatisfaction with in the past, it can help retain them for the long term.

**Comment by danielmackay** [comment]
Put yourself in the customers shoes and try to understand things form their point of view.  If an argument flairs up, I'd advise not trying to win it.  You may win the argument but lose the customer.

**Comment by jontygardner** [comment]
Solve the complaint with genuine care for the other person's interest and acknowledge the fault. And then tell your customer what practice you have implemented to fix this from happening again. 

E.g. We're sorry you slipped walking into the shop. To make sure this never happens again, we have our cleaners starting and finishing 1 hour early to make sure the floor is mopped and dry.

**Comment by vladislav-kir** [comment]
It is important to remember that you and your customer are the same team. Instead of taking the complaint personally, you should focus on solving the problem together.

**Comment by tkapa** [comment]
It's important to be accommodating but firm. Doing what you can but not allowing yourself to be pushed around.

**Comment by 2fernandez** [comment]
Manage client expectations from the start.
Overdeliver on what you've estimated, not under.
Listen to the client tentatively if you receive negative feedback, and promise to fix the issue as soon as possible.
Do not put negative feedback or correctable items on the backburner.

**Comment by DavidAbraham082** [comment]
Very important topic

**Comment by steven0x51** [comment]
Don't take it personally is a very good tip

**Comment by drwharris** [comment]
Always be open to ways to improve.

**Comment by JackDevAU** [comment]
Always a great topic to talk about. 
- Do not ignore the problem
- Listen to their complaint and picture it from their side 
- Fix the issue as soon as possible!

**Comment by KahaMason** [comment]
Not taking it personally is important. Understanding the problem and the customer's perspective can help bring everyone back on to the same page. Be understanding and accommodating.

**Comment by Harry-Ross** [comment]
I thought this was a really useful and informative rule - a very helpful guide to dealing with complaints.

**Comment by Hona** [comment]
Its important to stay cool headed & understand from the customer's perspectivity logically to help find a solution quickly

**Comment by jimmy1992abc** [comment]
Brillant topic, I've learned quite a lot from it. Especially, the value of "don't let them go" and solving the issue now. This is something I learned from Adam and what Adam is doing all the time.

Tips to handling complaints that you can share:
My personal experience: when there is a complaint, even if you cannot do anything, you need to tell the client that "it probably cannot, but I will try my best", then come back later and says "tried but cannot do it". Sometimes attitude is also important.

**Comment by christoment** [comment]
Good tip, I think having good customer service is one of the key for success 😄

**Comment by stephanfakossw** [comment]
Excellent topic!!! From my experience, it's important to jump on resolving client issues ASAP while communicating your circumstances (i.e. don't allow others to push you around).

**Comment by chrisschultzssw** [comment]
It's also important to reflect on how the complaint could have been avoided - maybe there's nothing you could have done differently, but it's worth thinking about.

**Comment by SamBWagner** [comment]
Always understand that clients are just humans and you should treat them as such. Work with them when you can, put up the boundaries when you should. Everything will be fine.

**Comment by landonmaxwell** [comment]
> Its important to resolve a client's complaint because studies have shown that: For every customer that complains, there may be 26 others that will not tell you they are dissatisfied
I would like to be able to read the studies this came from

**Reply by hveraus** [reply]
see - https://getmindful.com/blog/customer-service-stats-that-matter-part-ii/

**Comment by AttackOnMorty** [comment]
A complaint is a chance to turn a negative experience into a positive one.

**Comment by ozair-ashfaque** [comment]
Good tips to handle clients

**Comment by hveraus** [comment]
Wow, I think it's a good topic for marriages, you state how you’re feeling, be specific about behavior and then state what you need. Also, offer to make changes.

**Comment by JerryLuo5799** [comment]
Great rule, in my option, it means the client still wants to continue to cooperate if the client complains, so the most important things are what is your attitude and how quickly you can solve it. 
So don't try to argue with the client, try to solve his problem ASAP.

**Comment by sylhuang** [comment]
I actually found the number 26 mentioned in the rule astonishing, and wonder if we could add a link reference to the study? Interested to read the whole thing.
But this number really gets me to think handling complaints is much more important than I have imagined.

**Comment by AnkyChe** [comment]
Great rule, loves the point that "For every dissatisfied customer whose complaint you handle well: 95% will become loyal.They will recommend you to 5 others".

**Comment by jimmidier** [comment]
The methodology of Acknowledge, Apologize and Action is important as the guiding principal

**Comment by johnxu16** [comment]
Even if I have not received complaints from customers directly, staying calm and listening carefully sounds promising.

**Comment by JaneYangssw** [comment]
The rule indicates importance of maintaining customer relationship. And I'll try to handle clients' complaints following the rule in the future.

**Comment by KristenHu** [comment]
Great rule, actually if clients don't complain at all, we won't know if they're really satisfied. A complaint is a chance for us to make them genuinely delighted.

**Comment by SofieHong** [comment]
Great topic! 🌟
Keep in mind - Don't take it personally. ❗
Put a smile on your face, be positive, act immediately, and work with your clients to solve problems instead of being the opposite of each other.

**Comment by Berber-Mama** [comment]
Totally agree with the rule - keep calm, listen, empathise and deal with it. And don't forget to write an email when the problem is solved!

**Comment by zacharykeeping** [comment]
Good topic - it's important to stay positive and empathetic while taking control of the situation.

**Comment by evecogan** [comment]
Nothing like practice when it comes to taking complaints with grace 😇

**Comment by PothieuG** [comment]
Great topic!  
When dealing with complaints that are not personal in nature, it is important to take a step back and remain calm in every situation that involves anger or unhappiness.

**Comment by Freego1783** [comment]
Important topic !

Stay calm and focused on the problem. And don't hesitate to be honest when you don't know how to deal with a situation, but make sure you find someone who does.

**Comment by BrookJeynes** [comment]
Nice topic to cover, it's important to remain calm and not become defensive.

**Comment by duanxinhuan** [comment]
I like the topic and they all make sense.
It made me think about the days when Mike was there, he told me all those stuffs on the rule as well.

**Comment by TobyChurches** [comment]
Great Rule! Especially agree with not taking it personally 🙂

**Comment by rubycogan** [comment]
A super important skill, unfortunate that it's tough to follow if it doesn't come naturally to you. A great reminder for everyone that you can never be too good at people skills!

**Comment by leehawkinsssw** [comment]
Handling complaints well is something a lot of organizations still fail to do. I'm sure we all have first-hand experience of trying to be heard via call centres and other abstractions away from the people who could actually address our issues. As consultants dealing directly with clients, we're already in a good position to make them happier by directly addressing their concerns.

This topic reminded me of Jerry Weinberg's famous advice (from his seminal "The Secrets of Consulting" book): 

> "No matter how it looks at first, it's always a people problem." 

Listening to understand (rather than simply to respond) is the first step in handling a complaint well - let them have their say and explain why their problem is a problem _for them_ and resist the urge to react (rather than respond) by **deliberately taking pause**. 

The simple (but not easy) decision to actively listen to their complaint can really help take the heat out of the moment, most people are just not used to others actually listening to what they say.

**Comment by PennyWalker** [comment]
Unhappy customers are tough. You want to keep everyone happy, but you also have to make sure that SSW is fairly represented. I find that always actively listening to the problem and really making sure that they feel heard helps a lot. It's then important to come up with a plan, and also future-proof any improvements by documenting it somewhere & creating a standard where possible if something needed fixing. Try to follow up later to see if they're still happy, or need any further assistance.
